### PR TITLE
Fetch the API that contains fashion images

### PR DESCRIPTION
**Pull request by preciousonah**
**Description**

- Fetched an API and added a fashion search query that returns an object that contains the fashion images 
- Images are not displayed on the homepage yet - I'll map through the object and display it in my next pull request

**Milestones**

- Imported the pexels library in order to retrieve the API 
- Called an API that returns the images object 

Here's a preview of the webpage below: 
Image 7-6-23 at 6 50 PM

### CODE REVIEW

**Review by speltzka — COMMENTED** [review]
Why did you change all the quotation marks to ""? Is "" more correct that ''?

File: fashion-messaging-app-ui/src/components/App/App.jsx [hunk 1]
```diff
@@ -1,13 +1,35 @@
...
-  return (   
-    <div className="container">
-      <h1>Aesthetik.</h1>
-        <Search />
-        <Navbar />
-  </div>
-  )
-}
\ No newline at end of file
+const client = createClient("06ZcwjgbmJBM8T2TxLUZ5iwdXXGxiAgz0Z018b7QPKwR1ExipkFjaAuw");
```
Comment (speltzka) [inline fashion-messaging-app-ui/src/components/App/App.jsx]: What is this random string of characters? Can you define it as a variable?

File: fashion-messaging-app-ui/src/components/App/App.jsx [hunk 2]
```diff
@@ -1,13 +1,35 @@
-import './App.css'
+import "./App.css"
```
Comment (speltzka) [inline fashion-messaging-app-ui/src/components/App/App.jsx:1]: Why did you want to change the quotation marks for everything in this file? Not saying it's right or wrong, just wondering your reasoning?

File: fashion-messaging-app-ui/src/components/App/App.jsx [hunk 3]
```diff
@@ -1,13 +1,35 @@
...
-      <h1>Aesthetik.</h1>
-        <Search />
-        <Navbar />
-  </div>
-  )
-}
\ No newline at end of file
+const client = createClient("06ZcwjgbmJBM8T2TxLUZ5iwdXXGxiAgz0Z018b7QPKwR1ExipkFjaAuw");
+const query = "Fashion";
+const numberOfPhotos = 100;
```
Comment (speltzka) [inline fashion-messaging-app-ui/src/components/App/App.jsx]: I think it's better style-wise to put primitive constants in all caps. See here- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const

File: fashion-messaging-app-ui/src/main.jsx [hunk 4]
```diff
@@ -1,9 +1,9 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from "./components/App/App"
-import './index.css'
+import React from "react"
+import ReactDOM from "react-dom/client"
+import App from "./Components/App/App"
```
Comment (speltzka) [inline fashion-messaging-app-ui/src/main.jsx:3]: This may be where we changed the capitalization accidentally? Not sure

**Review by speltzka — CHANGES_REQUESTED** [review]
Hey Precious! I just added some small comments about capitalization, defining variables, etc. They are all small style things, your overall code is great! It would be awesome if you could amend this commit with the changes I requested. This will help us get used to the amending process

File: fashion-messaging-app-ui/src/components/App/App.jsx (hunk 3 above)
Comment (preciousonah) [reply fashion-messaging-app-ui/src/components/App/App.jsx]: Okay, got it!

File: fashion-messaging-app-ui/src/components/App/App.jsx (hunk 2 above)
Comment (preciousonah) [reply fashion-messaging-app-ui/src/components/App/App.jsx:1]: There were single quotes in the components I installed with Vite so I started with using single quotes but yesterday I realized i had switched to using double quotes down the line.
When I noticed this, I decided to change everything to double quotes for consistency

File: fashion-messaging-app-ui/src/components/App/App.jsx (hunk 1 above)
Comment (preciousonah) [reply fashion-messaging-app-ui/src/components/App/App.jsx]: That is the API key I fetched from pexels. I will define the variable name next time

**Comment by speltzka** [comment]
Looks good to me! Thanks for fixing the variable names

**Review by speltzka — APPROVED** [review]

**Pull request merged**